modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -672,13 +679,33 @@ public function testBanner(): void {
    *
    * @param \Symfony\Component\DomCrawler\Crawler $crawler
    *   The DomCrawler where to check the element.
+   * @param array $expected_values
+   *   The text values for paragraph fields.
    */
-  protected function assertBannerRendering(Crawler $crawler): void {
-    $this->assertEquals('Banner', trim($crawler->filter('.bcl-banner__content div')->text()));
-    $this->assertEquals('Description', trim($crawler->filter('.bcl-banner__content p')->text()));
-    $this->assertCount(1, $crawler->filter('svg.bi.icon--fluid'));
-    $this->assertStringContainsString('Example', trim($crawler->filter('a.btn')->text()));
-    $this->assertStringContainsString('#chevron-right', trim($crawler->filter('a.btn')->html()));
+  protected function assertBannerRendering(Crawler $crawler, array $expected_values = []): void {
+    $values = array_merge([
+      'title' => 'Banner',
+      'description' => 'Description',
+      'link_label' => 'Example',
+    ], $expected_values);
+    // Check optional title.
+    if (!empty($values['title'])) {
```

Comment:
empty() with `''` plays a bit weird as we all know. You are passing title = '', but it will actually do the empty scenario.
Let's clear this up: use NULL comparison, we need to pass NULL if we don't expect a title.

Reply:
Replaced